pkg/doc/verifiable/credential_test.go

```diff
@@ -95,7 +96,7 @@ const issuerAsObject = `
 }
 `
 
-func TestNew(t *testing.T) {
+func TestNewCredential(t *testing.T) { //nolint:funlen
```

Comment:
this function name doesn't seem to be long, there's no need for `nolint` here

Reply:
Right, Baha! It was longer at the previos version, now it's no need to use that.